var/spack/repos/builtin/packages/openmolcas/package.py

```diff
@@ -23,6 +23,8 @@ class Openmolcas(CMakePackage):
     depends_on('py-pyparsing', type=('build', 'run'))
     depends_on('py-six', type=('build', 'run'))
 
+    patch('CMakeLists.txt.patch')
```

Comment:
If the fix is for `aarch64` can we apply it only in that case?

Reply:
The conditions for applying patch files have been modified.